tools/test/conftest.py

```
@@ -0,0 +1,191 @@
+    if _p not in sys.path:
+        sys.path.insert(0, _p)
+
+# Load .env from repo root so ROCKETRIDE_APIKEY_* vars are available.
+# This must run before markers.py is imported (skipif conditions are
+# evaluated at import time).
+try:
+    from dotenv import load_dotenv
+
+    load_dotenv(_TOOLS_TEST.parent.parent / '.env')
+except Exception:
+    pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)